.github/workflows/integration.yaml

```diff
@@ -38,25 +38,22 @@ jobs:
           pip install tox
       - name: Run test
         run: tox -e unit
-        #  Integration Tests won't be executed for now as there are still some
-        #  issues with pytest-operator and k8s charms.
-        #
-        #  integration-test:
-        #    name: Integration tests
-        #    runs-on: ubuntu-latest
-        #    steps:
-        #      - name: Check out code
-        #        uses: actions/checkout@v2
-        #      - name: Setup Python
-        #        uses: actions/setup-python@v2
-        #        with:
-        #          python-version: 3.8
-        #      - name: Install Dependencies
-        #        run: |
-        #          pip install tox
-        #      - name: Setup operator environment
-        #        uses: charmed-kubernetes/actions-operator@master
-        #        with:
-        #            provider: microk8s
-        #      - name: Run test
-        #        run: tox -e integration -- -m container
+  integration-test:
+    name: Integration tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Install Dependencies
+        run: |
+          pip install tox
```

Comment:
We could use `apt install tox` here. I don't think there's a strong reason either way, other than that `apt` is less likely to surprise us with a new version that does things differently.

Reply:
We don't need to do either, actually, since [`actions-operator` already handles installing tox](https://github.com/charmed-kubernetes/actions-operator/blob/master/src/bootstrap/index.ts#L25).  Again, I think the `pip` vs `apt` is a hold-over from before `ubuntu-latest` was updated, and we could consider switching it (or pinning it in `actions-operator`).